TYPO3.Neos/Resources/Private/Styles/Media/_Media.scss

```diff
@@ -803,6 +803,11 @@
 		.neos-popover {
 			width: auto;
 			height: auto;
+			border-color: $grayMedium;
+
+			.neos-popover-title {
```

Comment:
I know we currently don't have any hard rules about how we write CSS, but could we perhaps start to move in the direction of having less nesting in CSS and towards somethings BEM-like. E.g. what's the sense of putting `.neos-popover-title` inside `.neos-popover`, where it already has this namespace inside its name?